RecoTauTag/RecoTau/src/AntiElectronIDMVA6.cc

```diff
@@ -1269,3 +1324,29 @@ double AntiElectronIDMVA6::dCrackEta(double eta)
 
   return std::abs(retVal);
 }
+
+bool AntiElectronIDMVA6::atECalEntrance(const reco::Candidate* part, math::XYZPoint &pos)
+{
+  bool result = false;
+  double bField = 3.8; // hopefully Teslas are proper units (better to take it from event setup) 
```

Comment:
Hi Slava, 

while trying to follow your suggestion, how do I get rid of this one (*) in my testing config?

Cheers,
Roger

(*)
[rwolf@lxplus000]~/work/scratch/TauPOG/miniAOD-tauMVAs/CMSSW_9_1_X_2017-04-24-1100/src% cmsRun RecoTauTag/RecoTau/test/rerunMVAIsolationOnMiniAOD.py
25-Apr-2017 01:12:38 CEST  Initiating request to open file file:patMiniAOD_standard.root
25-Apr-2017 01:12:39 CEST  Successfully opened file file:patMiniAOD_standard.root
rerunDiscriminationByIsolationMVArun2v1VLoose loadMVA = 1
rerunDiscriminationByIsolationMVArun2v1VTight loadMVA = 1
rerunDiscriminationByIsolationMVArun2v1VVTight loadMVA = 1
Begin processing the 1st record. Run 1, Event 251, LumiSection 6 at 25-Apr-2017 01:12:48.930 CEST
----- Begin Fatal Exception 25-Apr-2017 01:12:49 CEST-----------------------
An exception of category 'NoRecord' occurred while
   [0] Processing  Event run: 1 lumi: 6 event: 251 stream: 0
   [1] Running path 'p'
   [2] Calling method for module PATTauDiscriminationAgainstElectronMVA6/'rerunDiscriminationAgainstElectronMVA6'
Exception Message:
No "IdealMagneticFieldRecord" record found in the EventSetup for synchronization value
Run: 1 LuminosityBlock: 6 Event: 0 Time: 1
 Please add an ESSource or ESProducer that delivers such a record.
----- End Fatal Exception -------------------------------------------------
25-Apr-2017 01:12:49 CEST  Closed file file:patMiniAOD_standard.root